### ISSUE TITLE
Can´t install eventmachine-1.0.4 on Windows

### ISSUE THREAD

**Issue by fera2k** [eventmachine/eventmachine#555]
When I try to install eventmachine version 1.0.4 on my Windows machine, I get this error:

ERROR: Failed to build gem native extension.

Digging a little, I found on mkmf.log file that I must have some lib missing:
```
"gcc -o conftest.exe -IC:/Ruby21/include/ruby-2.1.0/i386-mingw32 -IC:/Ruby21/include/ruby-2.1.0/ruby/backward -IC:/Ruby21/include/ruby-2.1.0 -I. -DFD_SETSIZE=2048 -D_WIN32_WINNT=0x0501 -D__MINGW_USE_VC2005_COMPAT -D_FILE_OFFSET_BITS=64   -O3 -fno-omit-frame-pointer -fno-fast-math -g -Wall -Wextra -Wno-unused-parameter -Wno-parentheses -Wno-long-long -Wno-missing-field-initializers -Wunused-variable -Wpointer-arith -Wwrite-strings -Wdeclaration-after-statement -Wimplicit-function-declaration conftest.c  -L. -LC:/Ruby21/lib -lssl -lcrypto -lz     -lmsvcrt-ruby210  -lshell32 -lws2_32 -liphlpapi -limagehlp -lshlwapi  "
c:/ruby21/devkit/mingw/bin/../lib/gcc/i686-w64-mingw32/4.7.2/../../../../i686-w64-mingw32/bin/ld.exe: cannot find -lssl
c:/ruby21/devkit/mingw/bin/../lib/gcc/i686-w64-mingw32/4.7.2/../../../../i686-w64-mingw32/bin/ld.exe: cannot find -lcrypto
c:/ruby21/devkit/mingw/bin/../lib/gcc/i686-w64-mingw32/4.7.2/../../../../i686-w64-mingw32/bin/ld.exe: cannot find -lz
collect2.exe: error: ld returned 1 exit status
...
```
I could not find a solution. This libs (ssl, crypto, z) are available for Windows ? Where could I find then ? 

My Specs:
Windows 8.1 64 bits
Ruby 2.1.5 (32 bits) and DevKit-mingw64-32-4.7.2 (from RubyInstaller.org)

**Comment by fera2k** [comment]
After posting the issue, I tried some more digging, and finally could make it build!

Here the steps:

1) I found on this link: http://www.wittfella.com/openssl  a zip file containing the ssl and crypto libs already compiled for w32 and mingw.  Extracted the libs on C:\Ruby21\DevKit\mingw\i686-w64-mingw32\ folder (*.a  => \lib  and  include\openssl => \include)

2) I got the zlib file from the MinGW Soundforge repository here:
http://sourceforge.net/projects/mingw/files/MinGW/Base/zlib/
and downloaded the file: zlib-1.2.8-1-mingw32-dev.tar.lzma (opened with 7zip).
Inside it, there's a mingw32 folder with respective include and lib files.

3) gem install eventmachine -v 1.0.4 works perfectly.

I hope it helps somebody else!
